downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -39,7 +39,7 @@ enum class VcsHost(
     /**
      * The hostname of VCS host.
      */
-    protected val hostname: String,
+    val hostname: String,
```

Comment:
It was turned from `private `to `protected` in aa4d9c4b540a2ba053d8e0696f88d6356b593f1d as otherwise overridden functions would not have access to it, so it should stay public.